tests/integration/testdata/.gitignore

```diff
@@ -0,0 +1 @@
+/seccomp_syscall_test1
```

Comment:
as no one puts the binary in here, there's no need for this `.gitingore`.

Reply:
I did by accident, but okay.